utils/address.ts

```diff
@@ -1,6 +1,5 @@
 import { assertValidRailgunAddress } from '@railgun-community/quickstart';
 
 export const shortenAddress = (address: string) => {
-  assertValidRailgunAddress(address); // Will throw error on incorrect value
```

Comment:
This was causing an error in the form when typing an address. This validation will be handled by the form validation.